UI/TechTreeWnd.cpp

```diff
@@ -80,25 +81,38 @@ namespace {
         return true;
     }
 
-    struct ToggleCategoryFunctor {
-        ToggleCategoryFunctor(TechTreeWnd* tree_wnd, const std::string& category) : m_tree_wnd(tree_wnd), m_category(category) {}
-        void operator()() { m_tree_wnd->ToggleCategory(m_category); }
+
+    struct SetCategoryViewFunctor {
+        SetCategoryViewFunctor(TechTreeWnd* tree_wnd, const std::string& category) :
+                               m_tree_wnd(tree_wnd), m_category(category) {}
```

Comment:
indent initializer list one tab (4 spaces), not (necessarily) lined up with parameters.
preferably put each initializer on its own line, and the {} on a line after, indented to match the function name.